JDBC/README.md

```diff
@@ -5,6 +5,8 @@
 
 This application has been tested on MacOS, Linux, and Windows.
 
+The server must be configured for OAuth as described here: https://www.vertica.com/docs/11.1.x/HTML/Content/Authoring/Security/ClientAuth/OAuth/OAuthJDBC.htm
```

Comment:
I think it is easier to read if it is just linked from "here".  You can also replace the version with "latest" so you don't need to update it as often:
```
... described [here](https://www.vertica.com/docs/latest/HTML/Content/Authoring/Security/ClientAuth/OAuth/OAuthJDBC.htm).
```

Same applies to the link in the ODBC README.

Reply:
Changed.